### ISSUE TITLE
azurerm_cosmosdb_account does not handle capabilities correctly - always creates Mongo Server Version 3.6

### ISSUE THREAD

**Issue by sebhaub** [hashicorp/terraform-provider-azurerm#11294]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Terraform v0.14.8
hashicorp/azurerm v2.55.0

### Affected Resource(s)

* `azurerm_cosmosdb_account`

### Terraform Configuration Files

```hcl
resource "azurerm_resource_group" "foo" {
  name     = "foo"
  location = "West Europe"
}

resource "azurerm_cosmosdb_account" "bar" {
    name                = "bar"
    resource_group_name = azurerm_resource_group.foo.name
    location            = "West Europe"
    offer_type          = "Standard"
    kind                = "MongoDB"

    capabilities {
        name = "MongoDBv3.4"
    }
    
    geo_location {
        location          = "West Europe"
        failover_priority = 0
        zone_redundant    = false
    }

    consistency_policy {
        consistency_level = "Session"
    }
}
```
### Expected Behaviour

As i dont find anything else in the documentation, i expect with this setting a CosmosDB Database of kind MongoDB with server version 3.4 is created.

### Actual Behaviour

The ServerVersion is configured as 3.6

### Steps to Reproduce

1. `terraform apply`

**Comment by mrshiva** [comment]
having similar issues, but trying to deploy version 3.2 of API without any "capabilities" block and it worked previously, but right now it will create 3.6 always on provider versions:

2.20.0
2.40.0
2.56.0

Could you please help with this and provide guidance. Thank you!

**Comment by yupwei68** [comment]
Hi @sebhaub , thanks for opening this issue. Currently the service creates mongodb v3.6 by default. We'll enhance a `mongo_server_version` in `azurerm_cosmosdb_account` to permit to configure mongo server version recently. But the available version  is `3.2`, `3.6` and `4.0`.
PR https://github.com/terraform-providers/terraform-provider-azurerm/pull/11486
![image](https://user-images.githubusercontent.com/56525716/116510463-8bc0f800-a8f7-11eb-9742-e3b81818fb96.png)

**Comment by catriona-m** [comment]
Thanks for opening this issue @sebhaub. It looks like this has now been resolved by #11486 so this can now be closed. Thanks!
